datacenterlight/templates/datacenterlight/base.html

```diff
@@ -10,7 +10,7 @@
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <meta name="description" content="Data Center Light by ungleich">
         <meta name="author" content="ungleich GmbH">
-
+        {% cms_toolbar %}
```

Comment:
I moved the cms_toolbar before the title tag because without that the title tag can't be managed by the cms. There is a JS error (ReferenceError: CMS is not defined) that prevents loading of text plugin for the title.